tools/js-sdk-release-tools/src/common/rushUtils.ts

```diff
@@ -49,6 +49,17 @@ async function packPackage(packageDirectory: string, packageName: string, rushxS
     logger.info(`Pack '${packageName}' successfully.`);
 }
 
+async function ensurePnpmInstalled() {
```

Comment:
I think this check is unnecessary because we have already executed npm install -g pnpm in the initialization script.
https://github.com/Azure/azure-sdk-for-js/pull/34941/files#diff-48659f52f8b877bce024a10ede220148befe328d05834381607fb648ff695f3cR3
<img width="1543" height="135" alt="image" src="https://github.com/user-attachments/assets/998139be-5eea-46a9-8ce6-2fafc02d3220" />